solidity/src/Base7683.sol

```diff
@@ -57,7 +57,7 @@ abstract contract Base7683 is IOriginSettler, IDestinationSettler {
     // ============ Events ============
 
     event Filled(bytes32 orderId, bytes originData, bytes fillerData);
-    event Settle(bytes32[] orderIds, address[] receivers);
```

Comment:
I'm typically a stickler for keeping unrelated diffs in another PR, but i'll be lenient :P